.github/workflows/go.yml

```diff
@@ -0,0 +1,42 @@
+# This workflow will build a golang project
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-go
+
+name: Go
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
+
+jobs:
+
+  build:
+    runs-on: ubuntu-latest
+    
+    services:
+      mysql:
+        image: mysql:5.7
+
+        env:
+          MYSQL_ROOT_PASSWORD: root
+        ports:
+          - 3306:3306
+        options: --health-cmd="mysqladmin ping" --health-interval=10s --health-timeout=5s --health-retries=3
+        
+    steps:
+    - uses: actions/checkout@v3
+    
+    - name: Set up Go
+      uses: actions/setup-go@v3
+      with:
+        go-version: 1.19
+
+    - name: Build
+      run: go build -v ./...
+
+    - name: Test
+      run: |
+        export MYSQL_HOST=localhost
+        export MYSQL_PASSWORD=root
```

Comment:
I think you can use `env` for variables? https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#env

Reply:
The editor gives my error highlights when I use `env` but maybe it's just a problem with the syntax highlighting. I'll give it a try with `env`.